_maps/RandomRuins/SpaceRuins/anomaly_research.dmm

```diff
@@ -1451,6 +1456,8 @@
 "MJ" = (
 /turf/closed/wall/r_wall,
 /area/misc/anomaly_research)
+"MM" = (
+)
```

Comment:
you just added a new one

Reply:
what the hell is wrong with this ruin man

Reply:
Whar the hell

Reply:
Oh I missed a few spots thats why